### DISCUSSION TITLE
https://repo.saltproject.io is down

### DISCUSSION THREAD

**Discussion by vincent-olivert-riera** [saltstack/salt#67030] [General]
`$title`

**Comment by dafyddj** [comment]
Please take a look at other recent posts in this forum including the pinned post.

**Comment by max-arnold** [comment]
Check out the last four blog entries https://saltproject.io/blog/, or the salt-announce mailing list, or the Community Discord.

**Comment by vincent-olivert-riera** [comment]
I see. Thank you!
